scripts-dev/complement.sh

```diff
@@ -44,25 +44,33 @@ usage() {
 Usage: $0 [-f] <go test arguments>...
 Run the complement test suite on Synapse.
 
-  -f    Skip rebuilding the docker images, and just use the most recent
+  -f, --fast
+        Skip rebuilding the docker images, and just use the most recent
         'complement-synapse:latest' image
 
+  --build-only
+        Only build the Docker images. Don't actually run Complement.
+
```

Comment:
Do we want to document what happens when you have both `--fast` and `--build-only`? Not that you'd ever want to.

Reply:
I'll write that they conflict.